duckduckgo/Android
# Site permissions translations

Task/Issue URL: https://app.asana.com/0/0/1202913333162289/f

### Description

Add translations for all supported languages for new site permissions feature

### Steps to test this PR

_Site permissions screen_

- Install from this branch
- Change device language
- Go to Settings > Site Permissions
- [ ] Check screen is translated

### No UI changes
